torchmetrics/__init__.py

```diff
@@ -60,5 +60,6 @@
     RetrievalPrecision,
     RetrievalRecall,
 )
+from torchmetrics.text import WER  # noqa: F401 E402
 from torchmetrics.text import BLEUScore  # noqa: F401 E402
```

Comment:
this should probably just be one line
```suggestion
from torchmetrics.text import BLEUScore, WER  # noqa: F401 E402
```